scripts/data_format/format_control-freeC.pl

```diff
@@ -0,0 +1,46 @@
+#! usr/bin/perl
+use Getopt::Long;
+
+sub usage {
+        print STDERR << "USAGE";
+usage: perl $0 [options]
+options:
+        -P_value|P:            *p.value.txt file from control_freeC
+        -INFO|I:               *info.txt file from control_freeC
+        -biospecimens_id|bs:   Tumor Biospecimens ID
+        -help|h|?:             print help information
+
+e.g.:
+        perl $0
+USAGE
+        exit 0;
+}
+
+GetOptions(
+        "P_value|P:s" => \$p_value,
+        "INFO|I:s" => \$info,
+		"biospecimens_id|bs:s" => \$bs,
+        "help|h|?" => \$help,
+);
+
+die &usage() if (!defined $p_value || !defined $info || $help);
+
+
+open IF,"$info";
```

Comment:
Along gathering the ploidy, do we want to add code snippet to gather purity from the `Sample_Purity` row in *info.txt file as well?

Reply:
@kgaonkar6 Thank you for the comments.`Sample_Purity`  doesn't use in the consensus CNV . does it use somewhere for the analysis, I am not aware of

Reply:
We need it for histology bixu responsible columns right?

Reply:
I think the `normal_fraction | tumor_fraction| germline_gender_estimation| tumor_ploidy` in histology file, maybe I miss the Purity one

Reply:
ah you are right! The tumor purity is not from controlfreec, it's fractions from Theta2. Sorry!